tutorials/assetlib/what_is_assetlib.rst

```diff
@@ -25,6 +25,20 @@ Please note that the AssetLib is relatively young - it may have various pain poi
 and usability issues. As with all Godot projects, the code repository is available on `GitHub <https://github.com/godotengine/godot-asset-library>`_,
 where you can submit pull requests and issues, so please do not hesitate to visit it!
 
+Types of assets
+---------------
+
+Be aware that there are, broadly, two different types of assets you can post.
+
+* Assets labeled as "Templates" or "Projects" appear under the "Templates"
+  tab in the Godot project manager. These assets are standalone Godot
+  projects that should be able to run. ("Demos" also show up here, but
```

Comment:
I think anyone can upload an asset in the "Demos" category. Only select assets will have the "Official" support level though.